libs/net/entities/Action.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace TNO.Entities;
 
-[Cache("actions")]
+[Cache("actions", "lookups")]
```

Comment:
This ensures all these cache keys will get updated when this type of entity is updated/added/removed.